modules/ROOT/pages/editor-command-identifiers.adoc

```diff
@@ -38,69 +38,49 @@ tinymce.activeEditor.editorCommands.commands.exec;
 [[coreeditorcommands]]
 === Core Editor commands
 
-The commands on the following table are provided by the {productname} editor and do not require any plugins to be enabled.
+==== Supported browser-native commands
+
+The commands on the following table are provided by the {productname} editor and provide the same or similar functionality to the https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#commands[native browser commands].
```

Comment:
```suggestion
The commands in the following table are provided by the {productname} editor and provide the same or similar functionality to the https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#commands[native browser commands].
```

I feel like there's a word that should go before "native browser commands" too... "Respective"? I'm assuming we didn't implement all the native commands, so partly trying to imply it's a subset.